packages/frontend/src/components/pages/Club/Club.tsx

```diff
@@ -41,8 +41,10 @@ export const Club = () => {
               </tr>
             </thead>
             <tbody>
-              {clubs.map((club) => (
-                  <ClubRow club={club} refreshList={getAllClubs}/>
+              {clubs
+              .sort((a, b) =>new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime())
```

Comment:
It would be better to put the sort after obtaining the data in the useEffet because in future we probably want to put custom filtering/sorting and if you filter in the "view" part, we will always have the same sorting